zipkin-ui/test/component_data/trace.test.js

```diff
@@ -212,10 +212,10 @@ describe('convertSuccessResponse', () => {
       duration: '168.731ms',
       services: 2,
       depth: 2,
-      totalSpans: 2,
-      serviceCounts: [
-        {name: 'backend', count: 1, max: 111},
-        {name: 'frontend', count: 2, max: 168}
+      spanCount: 2,
+      serviceNameAndSpanCounts: [
+        {serviceName: 'backend', spanCount: 1},
```

Comment:
second most important change (to me) was removing dead data as it helps us understand what is needed by which screen